JeMPI_Apps/JeMPI_Linker/src/main/java/org/jembi/jempi/linker/BackEnd.java

```diff
@@ -466,4 +485,39 @@ public record EventLinkPatientToGidSyncRsp(
          LinkInfo linkInfo) implements EventResponse {
    }
 
+   public static ArrayList<Notification.MatchData> getCandidates(final PatientRecord patientRecord) throws RuntimeException {
```

Comment:
The function returns candidate match data notifications, so maybe we should use more appropriate name ? WDYT ?